twistlock/metadata.csv

```diff
@@ -0,0 +1,15 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Let's populate this file

Reply:
I did

Reply:
It was actually populated when you made that comment!